packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -154,7 +181,7 @@ Non-text status indicators (such as icons or status dots) should be paired with
 possible and should always be given a `title` to give context and meaning to the icon or shape.
 Check out the [icon usage guidelines](/components/icon/usage-guidelines) for more detailed accessibility information.
 
-## Variations
+## Status types
 
 ### Process status
 
```

Comment:
todo: add borderless Status Badges and default Status Badges to the "Process status" examples

Reply:
and also, sorry, forgot to mark this whole section as "Updated" in the doc: https://docs.google.com/document/d/1kDSWZrUBIR0ZrliM80_IjKCM8rMIDiISuxHTlQ-1xvs/edit?disco=AAABT1IWB7U

^ adds more text and a set of examples under "Process status"